### PR TITLE
ACQ-1501: migration from bower to npm

### PR DESCRIPTION
**Pull request by maguero**
### Description
Migrating from `bower` to `npm`.

**TLDR**: All `bower` dependencies (origami components) are moved from `bower.json` to `npm` (`package.json`) as a `peer-dependencies`  as migration guide ask. It also requires to add [scope packages](https://docs.npmjs.com/cli/v8/using-npm/scope) `@financial-times/` to each origami import/require. Finally removing all bower configuration.

### Ticket
https://financialtimes.atlassian.net/browse/ACQ-1501

### CODE REVIEW

**Review by ekocian — APPROVED** [review]

**Review by valeksiev — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -37,14 +37,25 @@
     "@babel/plugin-transform-runtime": "^7.11.0",
     "@babel/preset-env": "^7.11.0",
     "@babel/preset-react": "^7.10.4",
-    "@financial-times/dotcom-build-bower-resolve": "2.6.2",
     "@financial-times/eslint-config-next": "^3.0.0",
     "@financial-times/n-gage": "^8.2.0",
+    "@financial-times/o-buttons": "7",
```
Comment (AniaMakes) [inline package.json:42]: [Question] Here we have the Origami components twice - first in `devDependencies` and then in `peerDependencies`. The official Origami guides suggests the modules should be installed as peer dependencies. So **why are the modules listed here in devDependencies on top of peerDependencies**? Wouldn't that cause version mismatch at some point? Or the installation preferring to go more generic?

**Review by AniaMakes — COMMENTED** [review]
I have one question about the code which I would like an answer to before I officially approve.

This appears to be pretty straightforward replacement of imports and `init`s. Off that I also have another question - how did you find all the places the changes needed to be made? Did you run a repo-wide search for `o-` (or whatever)? Basically - have you made sure that you've made _all_ the changes needed?

File: package.json (hunk 1 above)
Comment (maguero) [reply package.json:42]: Good catch 🙌 . They should be only as `peerDependencies`. I'll be fixing this. Thanks.

Comment (maguero) [reply package.json]: 4a71dae3e599a98504e88c31858ae5e1419fec10

**Comment by maguero** [comment]
> I have one question about the code which I would like an answer to before I officially approve.
> 
> This appears to be pretty straightforward replacement of imports and `init`s. Off that I also have another question - how did you find all the places the changes needed to be made? Did you run a repo-wide search for `o-` (or whatever)? Basically - have you made sure that you've made _all_ the changes needed?

I have run a repo-wide search for `o-` to update each import/require line and added the expected scope package (`@financial-times/o-****`). Then I run the app (`make run`) and checked all the components loaded by Storybook.

Finally, I've linked this new version with next-subscribe and run the most important journeys locally to check the correct behaviours and styles of components and utils.
- Linked locally by npm link
- But also tested by a pre-release version => [v23.0.0-beta.3](https://github.com/Financial-Times/n-conversion-forms/releases/tag/v23.0.0-beta.3)

**Review by AniaMakes — APPROVED** [review]
Great work! Thank you for answering all my questions!

**Review by ekocian — APPROVED** [review]

**Pull request merged**